com/yadro/HWManager/Fan.interface.yaml

```diff
@@ -45,3 +45,9 @@ properties:
         - readonly
       description: >
           Index of Fan PWM line
+    - name: speed_limit_max
+      type: uint32
+      flags:
+        - readonly
+      description: >
+          Upper speed limit for PWM
```

Comment:
a unit and range specification would be nice here

Reply:
ok